static/elements/chromedash-gantt.js

```diff
@@ -0,0 +1,198 @@
+  static get properties() {
+    return {
+      feature: {type: Object},
+      stableMilestone: {type: Number},
+    };
+  }
+
+  constructor() {
+    super();
+    this.feature = {};
+    this.stableMilestone = null;
+    window.csClient.getChannels().then((channels) =>
+      this.stableMilestone = channels['stable'].version);
+  }
+
+  static get styles() {
+    return [
+      SHARED_STYLES,
+      css`
+      :host {
+        width: 200px;
+        margin-top: 8px;
+      }
+      ul {
+        display: block;
+        font-size: 12px;
+        vertical-align: top;
+        padding: 0;
+        margin: var(--content-padding-half);
+      }
+      li {
+        display: inline-block;
+        width: 80px;
+        padding: 4px 8px;
+      }
+      .header li {
+        text-align: center;
+      }
+      .bar {
+        background-repeat: no-repeat;
+        background-size: 210px;
+        background-position: 100px;
+      }
+      .without-ot {
+        background-image: url(/static/img/gantt-bar-without-ot.png);
+      }
+      .with-ot {
+        background-image: url(/static/img/gantt-bar-with-ot.png);
+      }
+      .platform {
+        padding-top: 15px;
+        vertical-align: top;
+      }
+      .diamond {
+        transform: rotate(45deg);
+        width: 36px;
+        height: 36px;
+        text-align: center;
+        margin: auto;
+        border: 2px solid white;
+      }
+      .dev_trial {
+        background: #cfe2f3ff;
+      }
+      .origin_trial {
+        background: #6fa8dcff;
+      }
+      .shipping {
+        background: #0b5394ff;
+        color: white;
+      }
+      .stable {
+        border-color: #444;
+      }
+      .diamond span {
+        font-size: 14px;
+        font-weight: bold;
+        display: table-cell;
+        transform: rotate(-45deg);
+        width: 34px;
+        height: 34px;
+        vertical-align: middle;
+      }
+      .offset_0 { padding-left: 0px; }
+      .offset_1 { padding-left: 8px; }
+      .offset_2 { padding-left: 16px; }
+      .offset_3 { padding-left: 24px; }
+    `];
+  }
+
+  renderDevTrial(milestone) {
+    if (!milestone) return nothing;
+    return html`
+      <div class="diamond dev_trial
+                  ${milestone == this.stableMilestone ? 'stable' : ''}"
+       title="First milestone with this feature available to developers"
+      ><span>${milestone}</span></div>`;
+  }
+
+  renderOriginTrial(milestone) {
+    if (!milestone) return nothing;
+    return html`
+      <div class="diamond origin_trial
+                  ${milestone == this.stableMilestone ? 'stable' : ''}"
+       title="First milestone with this feature enabled on specific origins"
+      ><span>${milestone}</span></div>`;
+  }
+
+  renderShipping(milestone) {
+    if (!milestone) return nothing;
+    return html`
+      <div class="diamond shipping
+                  ${milestone == this.stableMilestone ? 'stable' : ''}"
+       title="Milestone with this feature enabled by default"
```

Comment:
```suggestion
       title="Milestone with this feature enabled by default."
```

Reply:
Is it correct to use a period?  These phrases are not complete sentences.

Reply:
Not necessarily. This is hover text, right? What does your other hover text do? It should be consistent.

Reply:
The other hover text does not include periods.  E.g., title="Adds a new feature to the site"
I'll adopt all your wording changes.